### ISSUE TITLE
Different matching results with the same code and data source

### ISSUE THREAD

**Issue by Echowinde** [wannesm/LeuvenMapMatching#38]
I used pip command to install the package on my laptop and desktop respectively, and their versions are both 1.1.3. However, the following code can successfully match a route on my laptop, while the same code doesn't work on the desktop
```python
import panda as pd
import geopandas as gpd
import osmnx as ox

G = ox.load_graphml('./graph/hangzhou.graphml')
G_p = ox.project_graph(G, to_crs=4549)
nodes, edges = ox.graph_to_gdfs(G_p, nodes=True, edges=True)
map_con = InMemMap(name='HZnetwork', use_latlon=False, use_rtree=True, index_edges=True)
for node_id, row in nodes.iterrows():
    map_con.add_node(node_id, (row['y'], row['x']))
for node_id_1, node_id_2, _ in G_p.edges:
    map_con.add_edge(node_id_1, node_id_2)

matcher = DistanceMatcher(map_con, max_dist=500, max_dist_init=500, 
                          min_prob_norm=0.1,  non_emitting_length_factor=0.95,
                          obs_noise=50, obs_noise_ne=50, dist_noise=50,
                          max_lattice_width=20, non_emitting_states=True)

gps_data = pd.read_csv('./test_data.csv')
geo_data = gpd.GeoDataFrame(data, geometry=gpd.points_from_xy(data.lng, data.lat), crs=4326)
geo_data = geo_data.to_crs(4549)
path = list(zip(geo_data.geometry.y, geo_data.geometry.x))

states, _ = matcher.match(path, unique=False)
mmviz.plot_map(map_con, matcher=matcher, show_lattice=True,
               show_labels=True, show_matching=True, show_graph=False, filename=None)

"""
laptop result
states = [(314934753, 314934751),  (314934751, 5338584732), .......

desktop result
states = []
"""
````
All the inputs, params and codes are sure to be the same. I have tried to reinstall the package but it had no effects.

**Comment by csjiezhao** [comment]
you can set the paramter `use_rtree=False`

**Comment by wannesm** [comment]
You can also (temporarily) activate extended logging to see where the two versions differ. Based on the empty output my suspicion would be that the graph is not properly build up on the desktop.

```
import leuvenmapmatching as mm
import logging
import sys
mm.logger.setLevel(logging.DEBUG)
mm.logger.addHandler(logging.StreamHandler(sys.stdout))
```

**Comment by Echowinde** [comment]
Thanks for your reply!

**Comment by wannesm** [comment]
There was also an issue with using the rtree that may have caused your problem. It is now fixed in the new version (v1.1.4).
